compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -477,6 +479,7 @@ def IREEGPU_TargetWgpAttr : AttrDef<IREEGPU_Dialect, "TargetWgp"> {
     "DotProductOpsAttr":$dot,
     // MMAs available on this target. Heuristic MMA selection runs is order.
     "MMAOpsArrayAttr":$mma,
+    "ScaledMMAOpsArrayAttr":$scaled_mma,
```

Comment:
My comment is not direct to the PR, but I've been wondering if we can either make this kind of attribute optional, or have a custom parser/printer for this? It looks not good to me if we need to update so many files when we add such support; some files do not care the change. It looks like an issue in the attribute design to me.